[WIP] vim-patch:8.1.{1327,1347}

janlazo opened the pull request:
**vim-patch:8.1.1327: unnecessary scroll after horizontal split**
Problem:    Unnecessary scroll after horizontal split.
Solution:   Don't adjust to fraction if all the text fits in the window.
            (Martin Kunev, closes vim/vim#4367)
vim/vim@a9b2535

**vim-patch:8.1.1347: fractional scroll position not restored after closing window**
Problem:    Fractional scroll position not restored after closing window.
Solution:   Do restore fraction if topline is not one.
vim/vim@bd2d68c

This is missing some vim patches so I'll update the screen tests later.

janlazo committed:
vim-patch:8.1.1327: unnecessary scroll after horizontal split

Problem:    Unnecessary scroll after horizontal split.
Solution:   Don't adjust to fraction if all the text fits in the window.
            (Martin Kunev, closes vim/vim#4367)
https://github.com/vim/vim/commit/a9b2535f44f3265940a18d08520a9ad4ef7bda82

justinmk commented:
Looks like a bunch of screen tests are broken :( This is a welcome change, though...

janlazo committed:
vim-patch:8.1.1347: fractional scroll position not restored after closing window

Problem:    Fractional scroll position not restored after closing window.
Solution:   Do restore fraction if topline is not one.
https://github.com/vim/vim/commit/bd2d68c2f42c7689f681aeaf82606d17f8a0312f

janlazo commented:
Moved to https://github.com/neovim/neovim/pull/10004
